### PR TITLE
RDKTT-3356 Enhance Cryptography Interface tests

### PR DESCRIPTION
**Pull request by AbhishekMurali545**

### CODE REVIEW

File: Source/cryptography/tests/cryptography_test/InterfaceTests.cpp [hunk 1]
```diff
@@ -296,14 +326,51 @@ TEST(DH, Generate)
...
 {
     cg = WPEFramework::Cryptography::ICryptography::Instance("");
+    int len;
     if (cg != nullptr) {
-        vault = cg->Vault(cryptographyvault::CRYPTOGRAPHY_VAULT_NETFLIX);
+
+	for( int i= 1; i < argc; ++i )
+	{
+	    len = strlen(argv[i]);
+	    if ( (len == 9) && !strncmp( (const char*)argv[i], "--NETFLIX", len) )
```
Comment (sebaszm) [inline Source/cryptography/tests/cryptography_test/InterfaceTests.cpp:338]: Could we have all the switches consistent in letter case?

**Review by sebaszm — CHANGES_REQUESTED** [review]
Minor comment only.

File: Source/cryptography/tests/cryptography_test/InterfaceTests.cpp (hunk 1 above)
Comment (AbhishekMurali545) [reply Source/cryptography/tests/cryptography_test/InterfaceTests.cpp:338]: Could you elaborate on the comment. (NETFLIX, PLATFORM, DEFAULT) all are consistent in letter case. Are we talking about --testDefaultInstance , should it be changed to --TESTDEFAULTINTERFACE.

Comment (sebaszm) [reply Source/cryptography/tests/cryptography_test/InterfaceTests.cpp:338]: Indeed. But I would make them all lowercase. Thanks.

Comment (AbhishekMurali545) [reply Source/cryptography/tests/cryptography_test/InterfaceTests.cpp]: Updated. Please review. Thanks

Comment (sebaszm) [reply Source/cryptography/tests/cryptography_test/InterfaceTests.cpp]: Looks good.

**Review by sebaszm — APPROVED** [review]
Looks good!

**Pull request merged**